### ISSUE TITLE
mail parameter in example reaction.json is missing an option

### ISSUE THREAD

**Issue by bebob1** [reactioncommerce/reaction#2516]
Not a big issue, but still I want to share it.

### Expected behavior
When setting the mail parameter in reaction.json correctly the mail server should be working after starting reaction up and show a green dot in the dashboard.

The example reaction.json says:
```
            "mail": {
                "user": "",
                "password": "",
                "host": "",
                "port": ""
            },
```

### Actual Behavior
Settings are put correctly in the dashboard, but the red stays red. Obviously the "service" option is missing.  The "service" parameter (e.g. "service": "custom") has to be set in reaction.json additionally. 

Like this it works fine for me:
```
            "mail": {
                "service": "custom",
                "user": "",
                "password": "",
                "host": "",
                "port": ""
            },
```

### Versions
Reaction: 1.3.0

**Comment by spencern** [comment]
@jshimko Any idea if this was fixed in #2751 or any other email updates you've had recently?

**Comment by aanchirinah** [comment]
I have noticed it too @bebob1 , I will fix it.

**Comment by jshimko** [comment]
@spencern, no, I don't think I changed that any time recently.  Adding that `custom` option is definitely what we want in the reaction.json though, so maybe someone can sneak that into the next merge?

**Comment by jshimko** [comment]
The pre-fix alternative is after you load a reaction.json, you can go to the dashboard and select `custom` from the dropdown and click save.  That should accomplish the same thing.

**Comment by spencern** [comment]
Resolved by #3013
